beacon/sync/src/main/java/tech/pegasys/teku/beacon/sync/events/SyncStateTracker.java

```diff
@@ -130,7 +133,9 @@ public boolean unsubscribeFromSyncStateChanges(long subscriberId) {
 
   private void updateCurrentState() {
     final SyncState previousState = currentState;
-    if (headIsOptimistic) {
+    if (!executionLayerIsAvailable) {
+      currentState = SyncState.AWAITING_EL;
```

Comment:
not sure about this.
if we are syncing and EL goes down we set `AWAITING_EL` which was representing that our head is in sync but needs to be fully verified by the EL which is still syncing.

In this case we have a bed condition where EL is completely offline, so we might need another state.. have to think more about that..

Reply:
I was thinking about adding a new `SyncState` (`EL_OFFLINE` for example). Yet that can't be really reflected in the [Sync API](https://ethereum.github.io/beacon-APIs/#/Node/getSyncingStatus) since the response doesn't include such info